lambdas/cfn-set-bucket-tags/README.md

```diff
@@ -0,0 +1,89 @@
+# cfn-set-bucket-tags
+
+The `SetBucketTags` sets the OwnerEmail of a bucket
+
+Inventory of source code and supporting files:
+
+- set_bucket_tags - Code for the application's Lambda function.
+- events - Invocation events that you can use to invoke the function.
+- tests - Unit tests for the application code.
+- template.yaml - A template that defines the application's AWS resources.
+
+## Use in a Cloudformation Template
+Insert in your Cloudformation template a block similar to this as a sibling
+of the S3 bucket (called in this example `S3Bucket`) :
+```yaml
+  'Fn::Transform':
+    Name: "SetBucketTags"
+    Parameters:
+      BucketName: !Ref S3Bucket
+```
```

Comment:
It would help to have a more complete example so we can see the relationship between S3 and this transform.  I would suggest to have an example that includes the S3Bucket definition, transform and the expected transformed template that will be produced.

Reply:
The template is not transformed. I will add some explanatory text to make this clear.